docs/conf/cincinnati/2018/news/welcome.rst

```diff
@@ -0,0 +1,63 @@
+:template: {{year}}/generic.html
+
+.. post:: Apr 18, 2018
+   :tags: cincinnati-2018, website, cfp, tickets, open-help
+
+Announcing Write the Docs Cincinnati + Open Help Website, Call for Organizers, and CFP
+======================================================================================
+
+Welcome to the official launch of `Write the Docs Cincinnati + Open Help 2018 <http://www.writethedocs.org/conf/cincinnati/2018/>`_! We are excited to announce the first-ever collaboration between Write the Docs and Open Help, coming this August.
+
+Since 2011, the Open Help Conference in Cincinnati featured attendee-led discussions and sprints for a wide range of people working on open-source and community-based projects. This year, Write the Docs and Open Help are coming together to bring 5 days of talks, conversations, and doc sprints a little closer to home for our East Coast and Midwestern documentarians!
+
+Here's a quick overview of the schedule:
+
+* Day 1: Talks on all things technical documentation
+* Day 2: Unconference discussion and open space
+* Days 3-5: Documentation sprints and writing days
+
+Write the Docs Cincinnati + Open Help is scheduled for August 18-22, 2018 at `Union Hall <http://unionhallcincy.com/>`_ in the historic Over the Rhine neighborhood in `Cincinnati, Ohio <https://www.google.com/maps/place/Union+Hall/@39.109885,-84.515631,15z/data=!4m5!3m4!1s0x0:0x3caee5bfc65866b3!8m2!3d39.109885!4d-84.515631>`_.
+
```

Comment:
Feels like we should probably explain a bit more about the specific events. What is an unconference? What do you need to do to prepare for it? We should be able to spin up the "unconf" page on the website and link to it (similar to http://www.writethedocs.org/conf/portland/2018/unconference/), but it feels like we're doing something "more" with it here. 

@shaunix, do you have existing copy here that we can use here?

Reply:
We can link to one of our own Unconf/Writing Day topics (on a related note, we can/should probably have a generic-community version of those that's not linked to a specific conference)

Reply:
Thank you -- I added a little more description for each day. If we have generic pages for each day or conference-specific pages, I can add those links.